tests/unit/deployers/kfserving.py

```diff
@@ -0,0 +1,44 @@
+from unittest.mock import patch
+from fairing.deployers.kfserving.kfserving import KFServing
+
+# Test kfserving function with default_model_uri.
+def test_kfserving_default_model_spec():
+    kfsvc_name = 'test_kfservice'
+
+    with patch('fairing.deployers.kfserving.kfserving.KFServing.deploy', return_value=kfsvc_name):
+        kfsvc = KFServing(framework='tensorflow',
+                          default_model_uri='gs://kfserving-samples/models/tensorflow/flowers')
+        assert kfsvc.deploy("test") == kfsvc_name
```

Comment:
we need to test more than just the service name. Testing the output of generate_kfservice is useful here.

Reply:
Updated the unit test file to check more from generate_kfservice outputs. Thanks.